app/views/places/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1 align="center"> List of places where you training </h1>
+<% @places.each do |places| %>
```

Comment:
when we iterate over a collection, the block variable should be singular, as we refer to a single element

Reply:
OK